### DISCUSSION TITLE
群晖-alist单点登录问题

### DISCUSSION THREAD

**Discussion by cdtubear** [AlistGo/alist#4870] [General]
群晖里的sso服务器里自带的oidc服务器，怎样设置才能让alist里的oidc单点登录生效呢

**Comment by chenjuefei** [comment]
这我的配置，可供参考！
![PixPin_2024-05-11_16-33-03](https://github.com/alist-org/alist/assets/35056864/abcabd7a-55e8-428f-aa37-33907c181886)
![PixPin_2024-05-11_16-35-54](https://github.com/alist-org/alist/assets/35056864/cbee0c76-8abf-48f6-8ac9-b49c3210ddd5)

**Reply by xyl912158708** [reply]
你好，我这边会提示【无效的 SSO 客户端应用程序。请进入应用程序列表摂页面，并再次尝试添加应用程序。】
请问您是什么版本的alist呢？

**Reply by chenjuefei** [reply]
v3.34.0  不过我觉得和版本无关，还是填写的地址有问题。在群晖SSO的 "重定向URL" 中要填写alist的URL。在alist中的 “单点登录端点名称” 填写是群晖SSO的URL。再在alist的个人资料里，绑定登录。即可~

**Reply by FeranyDev** [reply]
大佬你是怎么配置的为什么我这样配置会提示```unknown endpoint: [/webman/sso/.well-known/openid-configuration]```
